fcrepo-kernel-impl/src/test/resources/containmentIndexTest.xml

```diff
@@ -21,4 +21,9 @@
     <bean id="containmentIndex" class="org.fcrepo.kernel.impl.ContainmentIndexImpl">
         <property name="dataSource" ref="containmentIndexDataSource"/>
     </bean>
+
+    <!-- ReferenceService index to test -->
+    <bean id="referenceIndex" class="org.fcrepo.kernel.impl.services.ReferenceServiceImpl">
+        <property name="dataSource" ref="containmentIndexDataSource" />
```

Comment:
You are using the `containmenIndexDataSource`??

Reply:
It's just the bean name, instead of creating a second bean to connect to the database. I can change the name to be more generic.